CSS/Homepage.css

```diff
@@ -23,3 +30,45 @@
     align-items: center;
     height: 80px;
 }
+
+/* question-one */
+.form__list {
+    display: flex;
+    flex-direction: column;
+    min-height: 50vh;
+    justify-content: space-between;
+}
+
+.title__question {
+    margin: 0px 35px 26px 35px;
+}
+
+.form__item {
+    margin: 0px 0px 15px 35px;
+}
+
+/*question-three*/
+.form__range {
+    margin: 0px 39px 20px 50px;
+}
+
+.slider {
+    width: 100%;
+    height: 10px;
+    background: grey;
```

Comment:
this color and measure does not go in the design

Reply:
hello, I have changed the color as it is in the design